packages/core/src/Cardano/Address/ScriptAddress.ts

```diff
@@ -0,0 +1,47 @@
+import { Address, AddressType } from './Address';
+import { HexBlob, InvalidStringError, OpaqueString, assertIsHexString } from '@cardano-sdk/util';
+import { isRewardAccount } from './PaymentAddress';
+
+export type ScriptAddress = OpaqueString<'ScriptAddress'>;
```

Comment:
I dont think we need this type, shared wallets should have PaymentAddress and RewardAddress (same as the regular wallet), this addresses can represent both pub key hash credentials and script hash credentials

Reply:
Removed on fac71bd